ivy/array/creation.py

```diff
@@ -79,6 +79,50 @@ def full_like(
             an array having the same shape as ``self`` and where every element is equal
             to ``fill_value``.
 
+        Functional Examples
+        -------------------
+
+        With int datatype:
+            >>> x = ivy.array([1,2,3])
+            >>> fill_value = 0
+            >>> ivy.full_like(x , fill_value)
+            >>> ivy.array([0, 0, 0])
+
+        Instance Method Examples:
+        -------------------
+            ivy.array([1, 1, 1, 1, 1, 1])
+            >>> fill_value = 0.000123
+            >>> x = ivy.array(ivy.ones(5))
+            >>> y = x.full_like(fill_value)
+            >>> print(y)
+            ivy.array([0.000123, 0.000123, 0.000123, 0.000123, 0.000123])
+
+
+        With ivy.Array input:
+            >>> x = ivy.array([1, 2, 3, 4, 5, 6])
+            >>> fill_value = 1
+            >>> y = x.full_like(fill_value)
+            >>> print(y)
+            ivy.array([1, 1, 1, 1, 1, 1])
+
+        With ivy.Container input
```

Comment:
we don't need container examples in array instance methods

Reply:
ohh got it